doc/api/packages.md

```diff
@@ -83,6 +85,41 @@ future-proof the package in case the default type of Node.js ever changes, and
 it will also make things easier for build tools and loaders to determine how the
 files in the package should be interpreted.
 
+### Modules loaders
+
+Node.js has two system for resolving a specifier and load modules.
+
+There is the CommonJS module loader:
+
+* It is fully synchronous.
+* It is monkey patchable.
+* When resolving a specifier, if no exact match is found, it will try to add
+  extensions (`.js`, `.json`, and finally `.node`).
+* It supports [folders as modules][].
+* It treats `.json` as JSON text files.
```

Comment:
I’m not sure what this means. How else would a `.json` file be treated?

Reply:
As JavaScript? I mean, `.wasm` is treated as JavaScript, so I think it's worth being explicit here ^^ also it's to match the ESM list below.

Reply:
Maybe just remove the JSON bullet point from both lists? I’m not sure we need it. The lists are focusing on the differences between the loaders, not the similarities, and they can both import JSON.

Reply:
ES loader currently cannot import `.json` files by default. And even if it did, I think we should still list JSON support here.

Reply:
Okay, I can’t think of better phrasing, this is fine for now.